source/packages/integration-tests/src/support/greengrass2_provisioning_hooks.ts

```diff
@@ -56,16 +56,13 @@ const templateBucket = process.env.PROVISIONING_TEMPLATES_BUCKET;
 const templatePrefix = process.env.PROVISIONING_TEMPLATES_PREFIX;
 
 async function teardown(world: unknown) {
-    // logger.debug(`\ngreengrass_provisioning_hooks: teardown: in:`);
-
-    logger.debug(`\ngreengrass_provisioning_hooks: teardown: in: ${JSON.stringify(world)}`);
-
     // service cleanup
     const templates = ['IntegrationTest'];
     for (const t of templates) {
         try {
             await templatesSvc.deleteTemplate(t, getAdditionalHeaders(world[AUTHORIZATION_TOKEN]));
         } catch (err) {
+            // logger.error(`teardown: deleteTemplate: ${JSON.stringify(err)}`);
```

Comment:
What severity of error is this? Should we leave these logs in?

Reply:
These are in integ tests, and some of them might be intended or ignore-able. All of the lines I added are above comments of `// swallow the error` so there was an intent not to log these before. In teardown, the presumption is that an early test failure might result in the thing not getting created and therefore it will error when it attempts to delete it. At the moment, I don't think it's worth overhauling to properly check if the resource was created and deleted without errors.